fastlane/.xcovignore

```diff
@@ -1,2 +1,4 @@
 # Exclude PayPal Interfaces
 - Sources/PayPal/PayPalUIInterfaces
+# Exclude SwiftUI Interface until a decision is made on UITesting Frameworks
+- Sources/PayPal/Buttons
```

Comment:
There is not an easy way to test SwiftUI views via unit tests. Adding this to pass test coverage until we decide to do either standard UI testing or snapshot testing. Once a UI testing plan is created, we should remove this.